readme.md

```diff
@@ -390,6 +390,22 @@ const useStore = create<State>(set => ({
 }))
 ```
 
+You can also use an `interface`:
+
+```tsx
+import { State as ZustandState } from 'zustand';
+
+interface State extends ZustandState {
+  bears: number
+  increase: (by: number) => void
+}
+
+const useStore = create<State>(set => ({
+  bears: 0,
+  increase: (by) => set(state => ({ bears: state.bears + by })),
+}))
+```
```

Comment:
It looks a bit long, how about this?

```tsx
import { State } from 'zustand'

interface BearState extends State {
  bears: number
  increase: (by: number) => void
}
```

Reply:
I actually had it like that, but thought it would be better to keep naming consistent (since `State` used to be defined by the user, while now it's a named import).

In any case, I've updated the example 😃

Reply:
It's up to developers to rename things, but in the readme we never add `Zustand` prefix, so it's more consistent.